forge/services/snapshots.js

```diff
@@ -51,21 +133,38 @@ module.exports.copySnapshot = async (app, snapshot, toInstance, { importSnapshot
 
         let newCredentials
         if (!decryptAndReEncryptCredentialsSecret) {
-            console.warn('Assuming credentials are not encrypted as no decryptAndReEncryptCredentialsSecret was passed')
-            newCredentials = app.db.controllers.Project.exportCredentials(oldCredentials, null, targetInstanceSecret)
+            console.warn(
```

Comment:
```suggestion
            app.log.warn(
```

Appreciate this was a pre-existing console.warn statement, but we should remove them when we see them in the backend as they will trip up the log parsing. Will look at adding a lint rule for this.

May also want to add some more context to the log statement otherwise no way of knowing who/what/where/why.